install-panel.sh

```diff
@@ -695,23 +708,22 @@ function letsencrypt {
     if [[ "$CONFIGURE_SSL" =~ [Yy] ]]; then
       ASSUME_SSL=true
       CONFIGURE_LETSENCRYPT=false
+      configure_nginx
```

Comment:
Just to be sure that you have handled all cases here (it looks to me like you have).  `configure_nginx` must be run when the user is using HTTP or just `ASSUME_SSL`. `configure_nginx` and then `letsencrypt` must be run when the user is using Let's Encrypt. And if the `letsencrypt` fails, and the user still opts in to use the `ASSUME_SSL`, then we need to re-run `configure_nginx`. Are all these cases handled? Does the `configure_nginx` function handle being re-run without machine wipe?

Reply:
It handles that alright. Just `curl` overwrites the files, `ln` links the same thing, `sed` replaces variables as before and `rm -rf` doesn't delete anything.